server/src/test/java/io/crate/integrationtests/LogicalReplicationITest.java

```diff
@@ -455,7 +455,6 @@ public void test_subscription_state_order() throws Exception {
                         subscriptionStates,
                         contains(Subscription.State.INITIALIZING,
                                 Subscription.State.RESTORING,
-                                Subscription.State.SYNCHRONIZED,
```

Comment:
Looks like this state will never be set anymore and can be removed?

Reply:
https://github.com/crate/crate/pull/12406